tests/ImageSharp.Tests/Formats/Jpg/JpegColorConverterTests.cs

```diff
@@ -41,54 +42,40 @@ public JpegColorConverterTests(ITestOutputHelper output)
 
         [Theory]
         [MemberData(nameof(CommonConversionData))]
-        public void ConvertFromYCbCrBasic(int inputBufferLength, int resultBufferLength, int seed)
+        public void FromYCbCrBasic(int inputBufferLength, int resultBufferLength, int seed)
         {
-            ValidateRgbToYCbCrConversion(
+            ValidateConversion(
                 new JpegColorConverter.FromYCbCrBasic(8),
                 3,
                 inputBufferLength,
                 resultBufferLength,
                 seed);
         }
 
-        private static void ValidateYCbCr(in JpegColorConverter.ComponentValues values, Vector4[] result, int i)
+        [Theory]
+        [MemberData(nameof(CommonConversionData))]
+        public void FromYCbCrVector(int inputBufferLength, int resultBufferLength, int seed)
```

Comment:
Doesn't have to happen in this PR, but for 1.1 we need a variant of this test that works with:

```C#
FeatureTestRunner.RunWithHwIntrinsicsFeature(
                RunTest,
                args,
                HwIntrinsics.DisableAVX2);
```

Reply:
What I would like to do is a single test which I can then pass through the runner. That will test all the versions then.

Reply:
Can you parameterize it to also exercise the Vector8 the Vector4 and the scalar path?

Reply:
Yep, since the call change the environmental paths like `COMPlus_EnableAVX2=0` we can do the following.

```
FeatureTestRunner.RunWithHwIntrinsicsFeature(
                RunTest,
                args,
                HwIntrinsics.AllowAll | HwIntrinsics.DisableAVX | HwIntrinsics.DisableSSE );
```

Reply:
How to get `Vector.IsHardwareAccelerated == false`?

Reply:
`COMPlus_FeatureSIMD=0` should do it

Reply:
Yep that's it. We have `HwIntrinsics.DisableSIMD` for that.